deployment/environment.go

```diff
@@ -300,14 +312,49 @@ func NodeInfo(nodeIDs []string, oc NodeChainConfigsLister) (Nodes, error) {
 			var cpk types3.ConfigEncryptionPublicKey
 			copy(cpk[:], b)
 
-			selToOCRConfig[sel] = OCRConfig{
+			ocrConfig := OCRConfig{
 				OffchainPublicKey:         opk,
 				OnchainPublicKey:          common.HexToAddress(chainConfig.Ocr2Config.OcrKeyBundle.OnchainSigningAddress).Bytes(),
 				PeerID:                    MustPeerIDFromString(chainConfig.Ocr2Config.P2PKeyBundle.PeerId),
 				TransmitAccount:           types2.Account(chainConfig.AccountAddress),
 				ConfigEncryptionPublicKey: cpk,
 				KeyBundleID:               chainConfig.Ocr2Config.OcrKeyBundle.BundleId,
 			}
+
+			var details chain_selectors.ChainDetails
+			switch chainConfig.Chain.Type {
+			case nodev1.ChainType_CHAIN_TYPE_APTOS:
+				cid, err := strconv.Atoi(chainConfig.Chain.Id)
+				if err != nil {
+					return nil, err
+				}
+				var exists bool
+				_, exists = chain_selectors.AptosChainIdToChainSelector()[uint64(cid)]
+				if !exists {
+					return nil, fmt.Errorf("aptos chain id %d not found", cid)
+				}
+				details, err = chain_selectors.GetChainDetailsByChainIDAndFamily(chainConfig.Chain.Id, chain_selectors.FamilyAptos)
+				if err != nil {
+					return nil, err
+				}
+			case nodev1.ChainType_CHAIN_TYPE_EVM:
+				evmChainID, err := strconv.Atoi(chainConfig.Chain.Id)
+				if err != nil {
+					return nil, err
+				}
+				_, err = chain_selectors.SelectorFromChainId(uint64(evmChainID))
+				if err != nil {
+					return nil, err
+				}
```

Comment:
This looks redundant, the result isn't used? We can call `GetChainDetailsByChainIDAndFamily` directly

Reply:
ah, yes, nice catch. thanks.